documentation-templates/wazuh/config.yml

```diff
@@ -1,28 +1,28 @@
 nodes:
   # Wazuh indexer nodes
   indexer:
-    - name: node-1
-      ip: <indexer-node-ip>
-    #- name: node-2
-    #  ip: <indexer-node-ip>
-    #- name: node-3
-    #  ip: <indexer-node-ip>
+    - name: indexer-1
+      ip: "<indexer-node-ip>"
+    #- name: indexer-2
+    #  ip: "<indexer-node-ip>"
+    #- name: indexer-3
+    #  ip: "<indexer-node-ip>"
 
   # Wazuh server nodes
   # If there is more than one Wazuh server
   # node, each one must have a node_type
   server:
-    - name: wazuh-1
-      ip: <wazuh-manager-ip>
+    - name: server-1
+      ip: "<wazuh-manager-ip>"
     #  node_type: master
-    #- name: wazuh-2
-    #  ip: <wazuh-manager-ip>
+    #- name: server-2
+    #  ip: "<wazuh-manager-ip>"
```

Comment:
Is there a reason why the variables are not numbered? i.e `<wazuh-server-2-ip>`

Reply:
Done https://github.com/wazuh/wazuh-packages/pull/1981/commits/504383b753f21c4d46c998fd6418a53081343f1d